ospo-book/content/en/01-chapter.md

```diff
@@ -92,24 +95,26 @@ Overall, gather input from stakeholders on these areas by asking the following q
   * How much OSS is already being used in the organization?
   * How would you define the 'open source culture' within your organization?
   * What are the organization's goals and objectives for using open source?
-  * How is OSS currently being used within the organization?
-  * How is OSS currently being created within the organization?
+  * How's OSS currently being used within the organization?
```

Comment:
```suggestion
  * How is OSS currently being used within the organization?
```
I think that using "How's" is much less common than the other contractions and sounds a bit strange. We should reflect this in the style guide.